### DISCUSSION TITLE
Dịch "bag-of-word"

### DISCUSSION THREAD

**Discussion by LongNguyen1984** [d2l-ai/d2l-vi#2553] [Terminology]
Hiện tại mình thấy từ bag-of-word được dịch thay bằng  từ "túi từ". Tuy nhiên như mình thấy từ này là một từ riêng và đã trở thành một thuật ngữ rất thông dụng trong ngành CS nên có lẽ ta nên giữ lại nguyên gốc từ này thay vì dịch ra. 
Tương tự như từ skip-gram ở đây ta không dịch để tạo sự nhất quán trong tài liệu.

**Comment by lkhphuc** [comment]
Mình không có ý kiến gì. Ping @davidnvq .

**Reply by quangvnai** [reply]
Em thấy từ **skip-gram** chưa tìm được cách dịch phù hợp. Tuy nhiên, từ **bag-of-words** khi dịch sang T.Việt thì lại không gây khó hiểu vì nghĩa khá tương đương. 
Đã có một số tài liệu lựa chọn cách dịch sang tiếng Việt như **túi từ**, **túi đựng từ**, v.v.. Ví dụ, [luận văn thạc sĩ của 1 sv](http://lib.uet.vnu.edu.vn/bitstream/123456789/789/2/KTPM_KhongBuiTrung_K20_Tom_Tat.pdf).

**Reply by thanhcsf** [reply]
Anh @LongNguyen1984 có phản hồi gì không ạ?

**Reply by LongNguyen1984** [reply]
Chỉ là vấn đề về tính nhất quán và mình ưu tiên để từ gốc hơn. Nếu đã được sử dụng thông dụng từ "túi từ" rồi thì ta tiếp tục dùng thôi. :)
